includes/admin/class-wc-admin-dashboard.php

```diff
@@ -214,7 +214,9 @@ private function status_widget_stock_rows() {
 			$lowinstock_count = get_transient( $transient_name );
 
 			if ( false === $lowinstock_count ) {
-				$lowinstock_count = (int) $wpdb->get_var(
+
+				$lowinstock_count_query = apply_filters(
+					'woocommerce_status_widget_lowinstock_count_query',
```

Comment:
Please change to `woocommerce_status_widget_low_in_stock_count_query`.

Reply:
@claudiosanches changed, Thanks!